backup_and_restore/replacing-failed-master.adoc

```diff
@@ -0,0 +1,33 @@
+[id="replacing-failed-master-host"]
+= Replacing a failed master host
+include::modules/common-attributes.adoc[]
+:context: replacing-failed-master-host
+
+toc::[]
+
+This document describes the process to replace a single etcd member. This procedure assumes that there is still an etcd quorum in the cluster.
+
+[NOTE]
+====
+If you have lost the majority of your master hosts, leading to etcd quorum loss, then you must follow the disaster recovery procedure to xref:../backup_and_restore/disaster_recovery/scenario-1-infra-recovery.adoc#dr-infrastructure-recovery[recover from lost master hosts].
+====
+
+To replace a single master host:
+
+* xref:../backup_and_restore/replacing-failed-master.adoc#restore-remove-failed-master_replacing-failed-master-host[Remove the member from the etcd cluster].
+
+* xref:../backup_and_restore/replacing-failed-master.adoc#restore-add-master_replacing-failed-master-host[Add the member back to the etcd cluster].
+
+// TODO: Confirm. But how can you use this to scale up if the procedure assumes that you're adding a member that had previously been removed?
```

Comment:
@retroflexer You mentioned that this procedure was only for adding back members that had been previously removed. Can you give me details on why this is the case? Is it just because it needs to have valid certs (and DNS entries as you mentioned in your comment of "If it is a brand new member, you will not have the requisite certificates and DNS entries etc. So, it perforce has to be a previous member.") If someone had taken care of the DNS entries/certs on a brand new host, could they use this procedure to add it to the etcd cluster?

Reply:
Yeah. It requires certificates and DNS entries. If they can take care of certs and DNS, then, we could use the replacing procedure.

Reply:
But trying to stay out of trouble by explicitly asking them to use it for adding back a previously failed member.

Reply:
@retroflexer Then I think we probably should remove this line about scaling up/down